packages/components/src/entities/SamplesAssayButton.tsx

```diff
@@ -4,7 +4,7 @@ import { PermissionTypes } from '@labkey/api';
 
 import { QueryModel } from '../public/QueryModel/QueryModel';
 
-import { AssayImportSubMenuItem } from '../internal/components/assay/AssayImportSubMenuItem';
+import { AssayImportSubMenuItem } from './AssayImportSubMenuItem';
```

Comment:
This is getting jostled around a bit. In my `fb_projectAssayData` branch I moved `EntityCrossProjectSelectionConfirmModal` out of the entities package so `AssayImporSubMenuItem` could use it, but I didn't check that this was used only within the entities package, so this is probably the better move.  Just be aware in case my merge goes first.